### PR TITLE
Add a couple of logging calls to monitor what is done.

### PR DESCRIPTION
**Pull request by jonathanrocher**
Add action logging calls for building a plot or exporting a dashboard. Calls done at the new `ACTION` logging level (25) defined in `app_common`.

Requires https://github.com/KBIbiopharma/app_common/pull/55

### CODE REVIEW

**Comment by jonathanrocher** [comment]
Trivial, self-merging.

**Comment by jonathanrocher** [comment]
Trivial: self-merging...

File: pybleau/app/io/dataframe_plot_manager_exporter.py [hunk 1]
```diff
@@ -192,6 +193,9 @@ def export(self):
             ui = self.edit_traits(kind="livemodal")
 
         if not self.interactive or ui.result:
+            msg = "Exporting plot content to {}.".format(self.export_format)
```
Comment (chamshop) [inline pybleau/app/io/dataframe_plot_manager_exporter.py:196]: I'd like to get in the habit of using `f` literal strings for these messages.

File: pybleau/app/model/dataframe_plot_manager.py [hunk 2]
```diff
@@ -362,6 +366,9 @@ def _add_new_plot(self, config, position=None, list_op="insert",
         if position is None:
             position = self.next_plot_id
 
+        msg = "Generating {} plot...".format(config.plot_type)
```
Comment (chamshop) [inline pybleau/app/model/dataframe_plot_manager.py:369]: As above, I'd like to use `f` literal strings for these messages.

**Review by chamshop — APPROVED** [review]
LGTM, with a couple suggestions.
I am accepting TravisCI testing instead of running locally. Will be doing this from now on once appveyor is in place.

**Comment by jonathanrocher** [comment]
Thanks for the comments. Old habits... Will merge when all CIs are happy.

**Pull request merged**